src/main/java/lombok.config

```diff
@@ -0,0 +1 @@
+lombok.log.fieldName=LOGGER
```

Comment:
Could we add a single-line comment here explaining how this play with and Lombok and how this will generate a `LOGGER` attribute instead of the default `log` one?

Reply:
I would prefer to remove it and replace all the calls with log instead if that’s ok.